lib/src/http/managers/soundboard_manager.dart

```diff
@@ -0,0 +1,194 @@
+      ..guilds(id: guildId.toString())
+      ..soundboardSounds();
+
+    final request = BasicRequest(route);
+
+    final response = await client.httpHandler.executeSafe(request);
+
+    final raw = (response.jsonBody as Map<String, Object?>)['items'] as List<Object?>;
+
+    final sounds = parseMany(raw, parse);
+
+    sounds.forEach(client.updateCacheWith);
+
+    return sounds;
+  }
+
+  @override
+  Future<SoundboardSound> create(SoundboardSoundBuilder builder, {String? auditLogReason}) async {
+    final route = HttpRoute()
+      ..guilds(id: guildId.toString())
+      ..soundboardSounds();
+
+    final request = BasicRequest(route, method: 'POST', auditLogReason: auditLogReason, body: jsonEncode(builder.build()));
+
+    final response = await client.httpHandler.executeSafe(request);
+
+    final soundboard = parse(response.jsonBody as Map<String, Object?>);
+
+    client.updateCacheWith(soundboard);
+
+    return soundboard;
+  }
+
+  @override
+  Future<SoundboardSound> update(Snowflake id, SoundboardSoundUpdateBuilder builder, {String? auditLogReason}) async {
+    final route = HttpRoute()
+      ..guilds(id: guildId.toString())
+      ..soundboardSounds(id: id.toString());
+
+    final request = BasicRequest(route, method: 'PATCH', auditLogReason: auditLogReason, body: jsonEncode(builder.build()));
+
+    final response = await client.httpHandler.executeSafe(request);
+
+    final soundboard = parse(response.jsonBody as Map<String, Object?>);
+
+    client.updateCacheWith(soundboard);
+
+    return soundboard;
+  }
+
+  @override
+  Future<void> delete(Snowflake id, {String? auditLogReason}) async {
+    final route = HttpRoute()
+      ..guilds(id: guildId.toString())
+      ..soundboardSounds(id: id.toString());
+
+    final request = BasicRequest(route, method: 'DELETE', auditLogReason: auditLogReason);
+
+    await client.httpHandler.executeSafe(request);
+    cache.remove(id);
+  }
+
+  /// Send a soundboard sound to a voice channel the user is connected to. [soundId] is the id of the soundboard sound to play,
+  /// while [sourceGuildId] is the id of the guild the soundboard sound is from. (Required to play sounds from different servers.)
+  Future<void> sendSoundboardSound(Snowflake channelId, {required Snowflake soundId, Snowflake? sourceGuildId}) async {
+    final route = HttpRoute()
+      ..channels(id: channelId.toString())
+      ..sendSoundboardSound();
+
+    final request =
+        BasicRequest(route, method: 'POST', body: jsonEncode({'sound_id': soundId.toString(), 'source_guild_id': (sourceGuildId ?? guildId).toString()}));
+
+    await client.httpHandler.executeSafe(request);
+  }
+}
+
+class GlobalSoundboardManager extends SoundboardManager implements ReadOnlyManager<SoundboardSound> {
+  GlobalSoundboardManager(super.config, super.client) : super(identifier: 'soundboard');
+
+  @override
+  PartialSoundboardSound operator [](Snowflake id) => throw UnsupportedError('Cannot index a global soundboard sound');
```

Comment:
Why?

Reply:
Because of `testManager`, since before `GuildSoundboardManager` didn't implement `Manager<T>` but `ReadOnlyManager<T>`.